ChangeLog.md

```diff
@@ -9,6 +9,9 @@ SPDX-License-Identifier: MIT
 <!-- Don't forget to update the gas/transaction costs tables in the
 README when a new version is released. -->
 
```

Comment:
I think that it would be nice to make a new release right after adding kathmandu support, so IMO it would be nice to update the costs as well as bump the version number in this PR

Reply:
I was going to do that in a separate follow-up PR ^^

Just pushed a few new commits to bump up the version number, update the costs, and update the instructions to generate the costs table (since the Kathmandu + lima faucets work differently now).

---

I also took the opportunity to add a "release checklist" to the PR template. Since every project has a slightly different release process, and it's hard to remember every single one, I've been adding these checklist to some of our int-dev projects, here are a few examples:
* [o-clock](https://github.com/serokell/o-clock/blob/master/.github/pull_request_template.md#-release-checklist)
* [xrefcheck](https://github.com/serokell/xrefcheck/blob/master/.github/pull_request_template.md#-release-checklist)

Let me know what you think.

Reply:
By the way, it seems the gas costs have gone up :eyes:

Reply:
the release checklist sounds good to me, I think you included everything needed there :ok_hand: 

I mean, I guess it's still possible to miss some things to update in the README, but those are all in the same place and making a more detailed checklist seems unpractical.

> it seems the gas costs have gone up

indeed, transaction costs seem to be the same, but gas has gone up considerably.

There is some difference in the resulting Michelson contract (aside from formatting, I mean), but at a quick look I don't think that they account for it.

I should mention that there is some variation between calls and that there might have been changes in the gas model too. TBH I don't think that these changes in costs are concerning, hence why I didn't look at these too deeply,